pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -819,6 +823,48 @@ class Build {
         return fileName
     }
 
+    /*
+    Run the Riscv version reader downstream job.
+    In short, we archive the build artifacts to expose them to the job and run ./java version, copying the output back to here.
+    See riscv_version_out.groovy.
+    */
+    def readRiscvVersionString() {
+        // Archive the artifacts early so we can copy them over to the downstream job
+        try {
+            context.timeout(time: buildTimeouts.BUILD_ARCHIVE_TIMEOUT, unit: "HOURS") {
+                context.archiveArtifacts artifacts: "workspace/target/*"
```

Comment:
I'm assuming this doesn't stop the normal archive at the end  which will still be needed to pick up the metadata

Reply:
Yes that happens regardless